VideoLocker/src/main/java/org/edx/mobile/module/serverapi/IResponse.java

```diff
@@ -0,0 +1,13 @@
+package org.edx.mobile.module.serverapi;
+
+import android.os.Bundle;
+
+/**
+ * Created by rohan on 2/6/15.
+ */
+public interface IResponse {
```

Comment:
And this for response? http://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/HttpResponse.html